static/admin.css

```diff
@@ -78,3 +78,4 @@ a:visited:not(.active-link) {
   align-items: center;
   justify-content: center;
 }
+
```

Comment:
No need to add this line ;). Just for semantics, I know it just added up after you deleted changes